pkgs/applications/misc/fitnesstrax/default.nix

```
@@ -0,0 +1,44 @@
+{ pkgs,
+  fetchFromGitHub ? pkgs.fetchFromGitHub,
+  stdenv ? pkgs.stdenv,
+  rustPlatform ? pkgs.rustPlatform,
+  wrapGAppsHook ? pkgs.wrapGAppsHook,
+}:
```

```suggestion
{ pkgs
, fetchFromGitHub
,  stdenv
,  rustPlatform
, wrapGAppsHook
}:
```

callPackage is going to fill this in properly already

---

Done, plus added the other formal dependencies.